.github/workflows/build.yml

```diff
@@ -33,7 +36,7 @@ jobs:
 
             - name: Check Code Generation
               run: |
-                  make generate
+                  make gen-all
```

Comment:
Let's keep this as `generate` for consistency with the upstream.

Reply:
That's a good point. The change was motivated by the naming conventions within our makefile. We can definitely change it back to stay in line with the contrib repo.

What do you think about about keeping both?
- The `generate` command has the logic
- The `gen-all` command is an alias for `generate`
- The github workflow runs `make generate`